### DISCUSSION TITLE
Nested App Shell layout

### DISCUSSION THREAD

**Discussion by pankajparashar** [mantinedev/mantine#4739] [Q&A]
Is it possible to have an AppShell layout inside of another AppShell layout?

**Comment by rtivital** [comment]
No, nested AppShells are not supported

**Comment by assman** [comment]
I have a use case for this actually, is it possible to tweak the AppShell to implement this behavior? What props would I need to change?

**Reply by rtivital** [reply]
Nested app shells are not supported, it is not possible, you need to build a custom component for that
